libraries/abstractions/common_io/test/test_iot_uart.c

```diff
@@ -63,7 +63,7 @@
 /* Globals values which can be overwritten by the test
  * framework invoking these tests */
 /*-----------------------------------------------------------*/
-uint8_t uctestIotUartPort = 0; /** Default UART port for testing */
+uint8_t uctestIotUartPort = 2; /** Default UART port for testing */
```

Comment:
Please do not modify the value here. It is supposed to be configured in ports/common_io/iot_test_common_io_internal.c. See below example for ST board.

https://github.com/aws/amazon-freertos/blob/master/vendors/st/boards/stm32l475_discovery/ports/common_io/iot_test_common_io_internal.c#L37

Reply:
That was from my PR, I neglected to revert this change when I was debugging. My apologies.